db/migrate/20220203164307_remove_shortened_quarantine_filters.rb

```diff
@@ -0,0 +1,41 @@
+class RemoveShortenedQuarantineFilters < ActiveRecord::Migration[6.1]
+  def up
+    ActiveRecord::Base.transaction do
+      # Migrate saved advanced filters
+      relevant_filters.each do |uf|
+        contents = JSON.parse(uf[:contents])
+        contents = migrate_advanced_filter_contents(contents)
+        uf.update!(contents: contents.to_json)
+      end
+
+      # Migrate advanced filters in saved export presets
+      relevant_export_presets.each do |uep|
+        config = JSON.parse(uep[:config])
+        contents = config.dig('data', 'patients', 'query', 'filter')
+        contents = migrate_advanced_filter_contents(contents)
+        config['data']['patients']['query']['filter'] = contents
+        uep.update!(config: config.to_json)
+      end
+    end
+  end
+
+  def down
+    # This migration is irreversible
+  end
+
+  def migrate_advanced_filter_contents(contents)
+    contents.select { |content| %w[seven-day-quarantine ten-day-quarantine].exclude?(content['filterOption']['name']) }
```

Comment:
Question, what happens if either of these filters is the only filterOption set?  I tested with just one and multiple, and the single instance leads to the saved filter just being blank.  Should we be deleting those ones entirely since there isn't really a use for a blank saved filter?

Reply:
yeah I thought about it and that's a good point, @hackrm would it make sense to remove the entire filter?

Reply:
@timwongj If the saved filter is only comprised of the shortened quarantine option, completely removing that saved filters sounds appropriate to me.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1203/commits/d8a51b855aacd1bc986973517d98f8abbde44328